projects/plugins/jetpack/_inc/lib/admin-pages/class.jetpack-react-page.php

```diff
@@ -80,13 +80,18 @@ public function add_page_actions( $hook ) {
 	}
 
 	/**
-	 * Remove the main Jetpack submenu as it is not needed since the user centric onboarding.
-	 * The admins can access the Jetpack Dashboard or My-Jetpack menus instead.
+	 * Remove the main Jetpack submenu if a site is in offline mode or connected.
+	 * At that point, admins can access the Jetpack Dashboard instead.
 	 *
 	 * @since 13.8
 	 */
 	public function remove_jetpack_menu() {
-		remove_submenu_page( 'jetpack', 'jetpack' );
+		if (
+			( new Status() )->is_offline_mode()
+			|| Jetpack::is_connection_ready()
+		) {
```

Comment:
This reverts the change in https://github.com/Automattic/jetpack/pull/43012/files